BridgeAppSDK/SBAActiveTask.swift

```diff
@@ -97,6 +99,16 @@ extension ORKTremorActiveTaskOption {
     }
 }
 
+extension ORKMoodSurveyFrequency {
```

Comment:
This enum is passed into RK to determine language for the questions. Either "today, I feel" or "this week, I feel".

Basically, I am imagining that there might be a case for only checking in once a week and wanted to support that.

Sent from my iPhone

> On Aug 16, 2016, at 5:37 PM, Erin-Mounts <notifications@github.com> wrote:
> 
> In BridgeAppSDK/SBAActiveTask.swift:
> 
> > @@ -97,6 +99,16 @@ extension ORKTremorActiveTaskOption {
> >      }
> >  }
> >  
> > +extension ORKMoodSurveyFrequency {
> Does RK have to know something about the scheduling of the mood survey?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>